6-AdvancedScenarios/4-hybrid-SPA/App/controllers/authController.js

```diff
@@ -0,0 +1,89 @@
+const msalInstance = require('../msal');
+require('dotenv').config();
+const url = require('url');
```

Comment:
```suggestion
const url = require('url');
```

Looks like this can be removed.

Reply:
done!